cspell.json

```diff
@@ -0,0 +1,104 @@
+{
+    "version": "0.2",
+    "ignorePaths": [],
+    "dictionaryDefinitions": [],
+    "dictionaries": [],
+    "words": [
+        "acks",
+        "actix",
+        "Aleksandr",
+        "allocs",
+        "archivals",
+        "backpressure",
+        "Banasik",
```

Comment:
whaat

Reply:
Ah I guess we're all mentioned in the NEPs directory

Reply:
Yeah, with the TODOs and stuff atleast..